core/templates/dev/head/components/state-editor/state-editor-properties-services/state-hints.service.ts

```diff
@@ -28,10 +28,24 @@ import { UtilsService } from 'services/utils.service';
   providedIn: 'root'
 })
 export class StateHintsService extends StatePropertyService {
+  private activeHintIndex: any;
```

Comment:
Here and elsewhere, avoid using the "any" type. Use the correct type from the get-go. (If you need help with this, please talk to @bansalnitish.)

Reply:
Replaced with `number`.